README.md

```diff
@@ -1,2 +1,55 @@
 # telesto-base
-Base image for telesto.ai models
+Base image for telesto.ai models.
+
+# Instructions
+
+`telesto-base` contains a pip-installable Python package and a Docker image, allowing you to
+easily package your models for telesto.ai competitions.
+
+## telesto-base module
+To install the module, you can simply use pip:
+```
+pip install git+https://github.com/telesto-ai/telesto-base.git
+```
+If you would like to use the latest not yet released version, you can install the one in the 
+`develop` branch.
+```
+pip install git+https://github.com/telesto-ai/telesto-base.git@develop
+```
+
+## The base image
+The base image contains the pre-installed `telesto-base` module. Your submissions will use this
+as a base, so you'll only have to worry about the algorithms and not the packaging. To use it
+locally, you can pull the image from Docker:
+```
+docker pull telestoai/model-api-base:stable
+```
+or
+```
+docker pull telestoai/model-api-base:latest
```

Comment:
Do you think experts will be interested in differences between versions of our Docker images? Why to mention `:latest`?